db/schema.rb

```diff
@@ -161,6 +161,7 @@
     t.integer  "question_id",    null: false, index: {name: "fk__course_assessment_answers_question_id"}, foreign_key: {references: "course_assessment_questions", name: "fk_course_assessment_answers_question_id", on_update: :no_action, on_delete: :no_action}
     t.string   "workflow_state", limit: 255, null: false
     t.datetime "submitted_at"
+    t.boolean  "correct"
```

Comment:
Add a comment in the migration saying the correctness is independent of the grade.